libs/base/ui/structure/collapsible/src/collapsible.component.ts

```diff
@@ -1,20 +1,22 @@
 import { ButtonSize, ButtonType } from '@spryker-oryx/ui/button';
 import { IconTypes } from '@spryker-oryx/ui/icon';
-import { I18nMixin, featureVersion } from '@spryker-oryx/utilities';
+import { I18nMixin, Size, featureVersion } from '@spryker-oryx/utilities';
 import { LitElement, TemplateResult, html } from 'lit';
 import { property, query } from 'lit/decorators.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import {
   CollapsibleAppearance,
   CollapsibleAttributes,
 } from './collapsible.model';
+import { collapsibleStyles } from './collapsible.styles';
 import { collapsibleBaseStyle } from './styles';
 
 export class CollapsibleComponent
   extends I18nMixin(LitElement)
   implements CollapsibleAttributes
 {
-  static styles = [collapsibleBaseStyle];
+  static styles =
+    featureVersion >= '1.4' ? collapsibleStyles : [collapsibleBaseStyle];
```

Comment:
Is this a new convention?

Reply:
kind of, but not yet formalised. Few things that cross my mind when making changes For this PR:
- the way we've implemented themes is not so convenient. Say you introduce a new theme, you'd loose all styles for components that are theme driven (e.g. storefront vs backoffice). Going forward, I like our themes to only influence CSS variables. We could still add styles per theme, but the basic components should work properly. 
- keep it simple. I rather have single css file per component, and not an array. Complex CSS is likely an indicator  that the complex is too large anyway.